en/user_guide/user.rst

```diff
@@ -0,0 +1,53 @@
+User, Role and Group
+====================
+
+Open Orchestra offers the possibility to manage different users with different rights.
+
+User Management
+---------------
+
+The users page allows to search, create, edit and delete users.
+
+At the creation, some fields are required :
+
+* **first Name**
+* **last Name**
+* **e-mail**
+* **username**
+* **password**
+
+Once the user is created, you can add his default language and his groups.
+
+Role Management
+---------------
+
+A role gives one type of authorization for a user.
+
+Roles contains:
+
+* **name**
+* **descriptions**: describe role in every languages to display in admin panel
+* **original/destination status**: optional parameters to sharp roles for a content types and define a `workflow`_.
+
+Some roles are already created.
+
+For example:
+
+* **ROLE_ACCESS_USER**:  grant access to users management
+* **ROLE_ACCESS_TREE_TEMPLATE**: grant access to templates management
+* **ROLE_ACCESS_KEYWORD**: grant access to keywords management
+* **ROLE_ACCESS_LOG**: grant access to logs view
+* **ROLE_ACCESS_REDIRECTION**: grant access to redirection management
+* **ROLE_ACCESS_ROLE**: grant access to role management
+* **ROLE_ACCESS_SITE**: grant access to websites management
+* **ROLE_ACCESS_STATUS**: grant access to status management
+
+Group Management
+----------------
+
+Each group has a name, a list of roles and is associated to only one website.
+Users in a group inherit rights defined in this group for the website associated.
+Using group is the simple way to give users some rights.
```

Comment:
``the simple way`` sounds also wrong to me.

You could use ``the recomanded way`` or ``the simpliest way``.

For the second guess, take care, because there is no other way to give roles to an user